core/src/main/java/com/yahoo/imapnio/async/internal/ImapAsyncSessionImpl.java

```diff
@@ -356,7 +356,7 @@ public <T> ImapFuture<ImapAsyncResponse> startCompression() throws ImapAsyncClie
     /**
      * @return true if channel is closed; false otherwise
      */
-    boolean isChannelClosed() {
+    public boolean isChannelClosed() {
```

Comment:
could you add unit test coverage for isChannelClosed in ImapAsyncSessionImplTest.java